frontend/components/MostViewed.js

```diff
@@ -29,7 +29,7 @@ const ListItem = styled('li')({
     minHeight: '4.5rem',
 });
 
-const Number = styled('span')({
+const NumberStyled = styled('span')({
```

Comment:
I didn't put too much thought into naming this `NumberStyled` as I know we'll be removing the styled component imminently.